src/Login/action_types.js

```diff
@@ -1,6 +1,4 @@
-export default {
-    USERNAME: 'LOGIN_USERNAME',
-    PASSWORD: 'LOGIN_PASSWORD',
-    REMEMBER: 'LOGIN_REMEMBER',
-    SUBMIT:   'LOGIN_SUBMIT'
-};
+export const USERNAME = 'LOGIN_USERNAME';
```

Comment:
When I do that, it doesn't even compile. In ES6, your exports must either be named or be the default export.